Library/Client.ts

```diff
@@ -234,10 +234,11 @@ class Client {
      */
     public track(
         data: ContractsModule.Contracts.Data<ContractsModule.Contracts.Domain>,
-        tagOverrides?: { [key: string]: string; }) {
+        tagOverrides?: { [key: string]: string; },
+        contextObject?: any) {
```

Comment:
I think we should make contextObject should be IContextProperties strongly types interface that we can add to typed definitions. The way it is, the signature of this object is unclear, so it is not very useful.